rfcs/text/0000-application-monitoring-for-ga.md

```diff
@@ -0,0 +1,146 @@
+### DCP PR:
+
+# System monitoring for General Availability
+
+## Summary
+
+This RFC describes the first standard revision for monitoring the distributed systems of the Human Cell Atlas (HCA) Data Coordination Platform (DCP). The standard is scoped to the minimum viable set of monitoring infrastructure the DCP will need to reach General Availability (GA). We anticipate that future revisions will be needed to meet our Service Level Objectives (SLOs) as DCP matures.
+
+The term monitoring is used here as defined in the [Google Site Reliability Engineering (SRE)](https://landing.google.com/sre/sre-book/chapters/monitoring-distributed-systems/) program. This document borrows heavily from SRE.
+
+## Author(s)
+
+* [Matthew Weiden](mailto:mweiden@chanzuckerberg.com)
+
+## Shepherd
+
+
+## Motivation
+
+Monitoring distributed systems [enables operators](https://landing.google.com/sre/sre-book/chapters/monitoring-distributed-systems/#why-monitor-pWsBTZ) to analyze long-term trends in system performance, compare experiments against a baseline, alert on failures, and conduct retrospective analysis on systems (debugging). These abilities are critical to keeping software correct and available.
```

Comment:
Please could you elaborate on what 'compare experiments against a baseline' means here.

Reply:
I mean "experiments" in the loose sense of the term: rough time series comparisons of changes to code and usage for diagnosing problems in the system. I do not intend to say that we should use a real-time monitoring to do A/B testing. That's better suited to an analytics and/or data warehousing framework.

Reply:
Sounds good. It might be worth doing a substitution like 'experiments' -> 'configuration or code changes' to avoid the overloaded meaning of scientific experiments.